db/migrate/20200721213935_create_user.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :password_digest
+      t.string :password_confirmation
```

Comment:
I didn't use a password_confirmation in any past projects. Is this something we need to save in the database?